app/GoogleReCaptchaWrapper.tsx

```diff
@@ -0,0 +1,28 @@
+"use client"
+
+import { GoogleReCaptchaProvider } from "@google-recaptcha/react"
+
+export default function GoogleReCaptchaWrapper({ children }: { children: React.ReactNode }) {
+  // This key is used by the frontend to load and execute the reCAPTCHA script.
+  const siteKey = process.env.NEXT_PUBLIC_RECAPTCHA_SITE_KEY!
+
+  if (!siteKey) {
+    console.warn("Missing NEXT_PUBLIC_RECAPTCHA_SITE_KEY in .env.local")
+  }
```

Comment:
Even if we don’t have a RECAPTCHA key, returning children lets things work easily in development.
```suggestion
  if (!siteKey) {
    console.warn("Missing NEXT_PUBLIC_RECAPTCHA_SITE_KEY in .env.local")
    return <>{children}</>
  }
```

Reply:
Thanks for the helpful note! Updated it to return children when the site key is missing, this makes development smoother.